### PR TITLE
Drop ApplicationQualifications#equivalency-details

### PR DESCRIPTION
**Pull request by CatalinVoineag**
## Context

This column is ignored and there's no data in production

## Changes proposed in this pull request

drop migration

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)
- [x] Inform data insights team due to database changes
- [x] Make sure all information from the Trello card is in here
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally
- [x] Add PR link to Trello card

### CODE REVIEW

**Comment by CatalinVoineag** [comment]
[Tech debt: Apply: remove unused equivalency_details field](https://trello.com/c/oCK5GGbt/1805-tech-debt-apply-remove-unused-equivalencydetails-field)

**Review by elceebee — APPROVED** [review]
Great to have this confusing column removed! Presumably you've informed data insights?

**Comment by CatalinVoineag** [comment]
> Great to have this confusing column removed! Presumably you've informed data insights?

Yep, they know

File: db/migrate/20240806133548_drop_equivalency_details.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class DropEquivalencyDetails < ActiveRecord::Migration[7.1]
+  def change
+    safety_assured do
```
Comment (dcyoung-dev) [inline db/migrate/20240806133548_drop_equivalency_details.rb:3]: Safety assured by ignoring column in #9629

**Review by dcyoung-dev — APPROVED** [review]

**Pull request merged**